src/features/Chat/Chat.module.scss

```
@@ -185,13 +190,19 @@
 
   &__input {
     flex-shrink: 0;
-    padding: 16px 20px 20px 20px;
+    // 모바일에서 입력영역 패딩 축소
```

Stylelint (17.4.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)